src/restclient/backup/backupservice.ts

```diff
@@ -76,7 +81,16 @@ export class BackupService extends RestService {
 
 	private handleError(e: Error): unknown {
 		if (e instanceof NodeRPCException) {
-			throw e;
+			switch (e.getCode()) {
+				case NodeRPCException.UNAUTHORIZED:
+					throw new UnauthorizedException(e.message, e);
+				case NodeRPCException.BAD_REQUEST:
+					throw new InvalidParameterException(e.message, e);
+				case NodeRPCException.NOT_FOUND:
+					throw new NotFoundException(e.message, e);
+				default:
+					throw new ServerUnknownException(NodeRPCException.SERVER_EXCEPTION, e.message, e);
```

Comment:
Please don't update the handleError method. If e is a NodeRPCException, it has already been parsed by the HttpClient using NodeRPCException.forHttpCode(...). That's why we simply rethrow the error in this case.